### PR TITLE
Add logger for sinatra apps

### PR DESCRIPTION
**Pull request by robinbortlik**
Adds logging support for Sinatra applications. Code is mostly inspired by Grape integration which is already implemented.

### CODE REVIEW

File: lib/sapience/sinatra.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+module Sinatra
+  class Base
+    extend Sapience::Descendants
+  end
+end
+
+module Sapience
+  class Siantra
+    Sapience.configure
```
Comment (jelamo) [inline lib/sapience/sinatra.rb]: typo :)  Siantra --> Sinatra

Comment (robinbortlik) [reply lib/sapience/sinatra.rb]: Thanks for CR @jelamo , happy to see that you are missing Ruby :)

Comment (jelamo) [reply lib/sapience/sinatra.rb]: oh was that Ruby? then leave the type what need fixing is the language :D

Comment (robinbortlik) [reply lib/sapience/sinatra.rb]: That is a good one :D

**Pull request merged**